server/entity/splashable.go

```diff
@@ -10,6 +10,20 @@ import (
 	"time"
 )
 
+// SplashableBlock is a block that can be splashed with a splash bottle.
+type SplashableBlock interface {
+	world.Block
+	// Splash is called when a type that implements splashable splashes onto a block.
```

Comment:
```suggestion
	// Splash is called when a water bottle splashes onto a block.
```

and same for SplashableEntity